docs/custom-instrumentation.asciidoc

```diff
@@ -1,14 +1,85 @@
 [[custom-instrumentation]]
 === Custom instrumentation
 
-When installed ElasticAPM will automatically wrap your app's request/responses in transactions and report its errors.
+When installed ElasticAPM will automatically wrap your app's request/responses in
+transactions and report its errors.
+It also wraps each background job if you use Sidekiq or DelayedJob.
```

Comment:
Generally it is not really clear to me from the description which Transactions, Spans and Errors are captured automatically for which Frameworks and Libraries, and which ones does a user need to manually collect.

You say 

> When installed ElasticAPM will automatically wrap your app's request/responses... 

I'd change this to `When installed and properly configured (link to how to set it up) ... `